src/app/utils/plug-descriptions.ts

```diff
@@ -204,6 +205,14 @@ function getPerkDescriptions(
       usedStrings.add(notif);
     }
   }
+  function addSpecialDescriptions() {
+    if (plug.displayProperties.name.includes('Harmonic')) {
+      results.push({
+        perkHash: -usedStrings.size,
```

Comment:
Should probably add this description to `usedStrings` too so that `perkHash` can remain a unique key.